doc/BACKUP.md

```diff
@@ -394,14 +394,16 @@ it just gains the option to use a PostgreSQL database as well.
 If you just want to use PostgreSQL without using a cluster (for
 example, as an initial test without risking any significant funds),
 then after setting up a PostgreSQL database, you just need to add
-`--wallet=postgresql://${USER}:${PASSWORD}@${HOST}:${PORT}/${DB}`
+`--wallet=postgres://${USER}:${PASSWORD}@${HOST}:${PORT}/${DB}`
 to your `lightningd` config or invocation.
 
 To set up a cluster for a brand new node, follow this (external)
 [guide by @gabridome][gabridomeguide].
 
 [gabridomeguide]: https://github.com/gabridome/docs/blob/master/c-lightning_with_postgresql_reliability.md
 
+(plain url in case of broken link on RTD: https://github.com/gabridome/docs/blob/master/c-lightning_with_postgresql_reliability.md).
+
```

Comment:
Hm, not very satisfactory to have the URL not be linked at all. Any change a URL shortener would be acceptable and we could link directly in line 403 then?